src/app.ts

```diff
@@ -1,47 +1,19 @@
-require("dotenv").config()
```

Comment:
@broarr Do you have an alternative way of loading .env files from the package.json directly? We use this for non-docker local testing when needed.

Reply:
I moved this line of code over to `src/index.ts` and changed the entrypoints for the server to `src/index.ts`